test/cypress/plugins/index.js

```diff
@@ -11,16 +11,10 @@
 // This function is called when a project is opened or re-opened (e.g. due to
 // the project's config changing)
 
-// const browserify = require('@cypress/browserify-preprocessor')
+const browserify = require('@cypress/browserify-preprocessor')
 
-// module.exports = (on) => {
-//   // Make cypress share the same babelrc config as the rest of the project
-//   // https://github.com/cypress-io/cypress-browserify-preprocessor#modifying-default-options
-//   const options = browserify.defaultOptions
-//   options.browserifyOptions.transform[1][1].babelrc = true
-
-//   on('file:preprocessor', browserify(options))
-// }
+const options = browserify.defaultOptions
+options.browserifyOptions.transform[1][1].babelrc = true
```

Comment:
Why was this uncommented if it wasn't needed before?

Reply:
Actually, I didn't remove the comment, just moved it below. Please check the line 26, 27.
However, I can make it as before so that we have less changes for this solution.

Reply:
@Silver-IT 

> Actually, I didn't remove the comment, just moved it below

What do you mean...? The code has been uncommented is what I meant. It's now using browserify when I think it didn't need to use it?

Reply:
You mean
- `browserify` is not needed now?
- These changes should be reverted again?

Reply:
I am saying that before we had `browserify` disabled here by having it commented out, right? So if it was working fine without `browserify` enabled, then there's no need to enable it because it will slow things down.

Reply:
> I am saying that before we had browserify disabled here by having it commented out, right?

Yes

> So if it was working fine without browserify enabled, then there's no need to enable it because it will slow things down.

I agree with you. Thanks